src/test/local-shibbo/README.md

```diff
@@ -0,0 +1,27 @@
+## Start
```

Comment:
Voisi siirtää nämä `local-shibbo`-resurssit `main`in alle kun nehän eivät suoranaisesti liity testeihin vaan ovat vain toinen tapa käynnistää/ajaa sovellusta